### ISSUE TITLE
Feature request, large font size for vim command

### ISSUE THREAD

**Issue by norogoth** [VSCodeVim/Vim#7817]
Currently the vim commands are very small because they display in the status bar. The only way to make this font bigger is to increase everything, which makes the whole UI more cluttered
![small_font](https://user-images.githubusercontent.com/52639031/177358888-e6cb2586-02d8-42cf-b1db-f25d000ba58f.png)
It would be nice if there is an option to have vim commands show up bigger so you can more easily see what commands you are running.
![big_font](https://user-images.githubusercontent.com/52639031/177358883-6c184f2d-44f0-4462-802f-52a952dd2c62.png)

Thanks for making the extension! It is very useful to many people!

**Comment by clscott** [comment]
I agree that this would be great, especially if the command were centered in the focused editor and semi transarent

**Comment by norogoth** [comment]
Looks like this issue got closed and the related one mentioned here. Did they both get closed because it was thought that the other one would remain open or is this feature idea shelved?

**Comment by jaccohub** [comment]
+1

I currently have to increase the zoom level to 0.6~ to even be able to read this text. This affects the entire editor so its not optimal, but at least it's better than the defaults for me.

**Comment by lukas-schweter** [comment]
+1

**Comment by n1rdeks** [comment]
i see, nothing happend here. Just close this issue. 

Maybe u watch for example, how fix this problem neovim extension? I know, VSCode to long fix customize statusbar (I create or support issue, I forgot). So, maybe need add popup window for command?

**Comment by damonholden** [comment]
would be nice to see something done about this.

**Comment by akivagold** [comment]
As a workaround, I use [Custom UI Style](https://marketplace.visualstudio.com/items?itemName=subframe7536.custom-ui-style) with the following setting:
```json
"custom-ui-style.stylesheet": {
        ".statusbar": "display: flex",
        ".statusbar": "font-size: 16px !important;",
}
```
It will make all the text in the status bar bigger
